app/controllers/posts_controller.rb

```diff
@@ -58,7 +77,11 @@ def set_board
       @board = Board.find(params[:board_id])
     end
 
+    def set_user
+      @user = current_user
```

Comment:
Skoro już o tym rozmawialiśmy, to User.find(current_user.id) jest lepsze, o czym sam się przekonałeś